INSTALL.md

```diff
@@ -0,0 +1,92 @@
+#Installation for Lnd and Btcd
+
+### If Glide isn't installed, install it:
+```
+$ go get -u github.com/Masterminds/glide
+```
+
+###Install lnd:
+```
+$ cd $GOPATH
+$ git clone https://github.com/lightningnetwork/lnd $GOPATH/src/github.com/lightningnetwork/lnd
+$ cd lightningnetwork/lnd
+$ glide install
+$ go install . ./cmd/...
+```
+
+###Create lnd.conf:
+**On MacOS, located at:**
+/Users/[username]/Library/Application Support/Lnd/lnd.conf
+
+**On Linux, located at:**
+~/.lnd/lnd.conf
+
+**lnd.conf:**
+```
+[Application Options]
+rpcuser=kek
+rpcpass=kek
+btcdhost=127.0.0.1
+debuglevel=trace
```

Comment:
The logs can get rather spammy with a full `trace` setting applied, so I think instead we should make the default here (at least for the docs), be `debug`.